Gruntfile.js

```diff
@@ -1,84 +1,117 @@
 module.exports = function (grunt) {
   grunt.initConfig({
-    pkg: grunt.file.readJSON('package.json'),
+    pkg: grunt.file.readJSON('./package.json'),
 
     clean: {
       options: {
         force: true
       },
-      all: ['dist/']
+      all: ['./dist/', './www/']
     },
 
     fileExists: {
-      css: ['dist/css/winstrap.css']
+      css: ['./www/css/winstrap.min.css']
     },
 
     sass: {
       options: {
-        outputStyle: 'nested',
+        outputStyle: 'compressed',
         sourceMap: true,
         precision: 5,
         includePaths: [
-            "bower_components"
+            "node_modules"
         ]
       },
       dist: {
         files: {
-          'dist/css/winstrap.css': 'src/scss/winstrap.scss'
+          './dist/css/winstrap.min.css': './src/scss/winstrap.scss',
+          './dist/css/winstrap.css': './src/scss/winstrap.scss'
+        }
+      }
+    },
+    
+    //  Uglify winstrap.js
+    uglify: {
+      winstrapjs: {
+        options: {
+          beautify: true
+        },
+        files:{
+          './dist/js/winstrap.js': [

```

Comment:
Hey @edrohler, thanks for making this change!

Just one thing, I don't think winstrap.js should be put into /dist. It just contains some js needed by the documentation site, which is why we called it app.js instead of winstrap.js. Can you please rename it to app.js and move it into /www?